jekyll/_cci2/circleci-images.md

```diff
@@ -93,37 +106,93 @@ The following packages are installed with `apt-get` on every image:
 - [xvfb](https://packages.debian.org/stretch/xvfb)
 - [zip](https://packages.debian.org/stretch/zip)
 
-### Other Packages
-
 The following packages are installed via `curl` or other means:
 
 - [Docker client](https://docs.docker.com/install/)
 - [Docker Compose](https://docs.docker.com/compose/overview/)
 - [dockerize](https://github.com/jwilder/dockerize)
 - [jq](https://stedolan.github.io/jq/)
 
-## Image Variants
+## Customizing a Convenience Image
+
+You can customize convenience images by adding image tags.
+There are three good reasons to do this:
+
+- You want to pin an image to a certain version or operating system (OS).
+- You want to specify a particular instance of an image with a SHA.
+- You want to modify an image by installing additional tools
+or changing the behavior of the image.
 
-CircleCI maintains variants of convenience images. These can be created by adding optional suffixes to the end of image tags.
+### Pinning an Image Version or OS
 
-**Note:** If you choose to use the `latest` tag, the image may change unexpectedly and create surprising results.
+Since convenience images are based on the **latest** versions of upstream images,
+it is best practice
+to use the most specific image possible.
+This prevents the upstream image
+from introducing unintended changes to your image.
 
-### Language Image Variants
+For example,
+instead of `circleci/golang:latest`,
+consider using `circleci/golang:1.8.6-jessie`.
+Since the second image specifies both the version and OS,
+it is less likely to change unexpectedly.
 
-- `-node`: includes Node.js for polyglot applications
-- `-browsers`: includes Java 8, PhantomJS, Firefox, and Chrome
-- `-node-browsers`: a combination of the `-node` and `-browsers` variants
+See below for a list of the [Latest Image Tags by Language](#latest-image-tags-by-language).
 
-### Service Image Variants
+**Note:**
+Excluding the [variants below](#changing-the-tools-and-behavior-of-an-image),
+CircleCI does not control which tags are used.
+These tags are chosen and maintained by upstream projects.
+Do not assume
+that a given tag has the same meaning across images!
 
-- `-ram`: variants that use the RAM volume to speed up builds
+### Specifying a Past Image Build With a SHA
 
-<hr>
+You can also use a SHA
```

Comment:
We could use the proper term, Docker image digest, and perhaps link to Docker's Doc on it so that we don't have to explain it here?